module-1/pandas-project/your-code/shark-attack.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
1. In[7]: you can do a function with for loop over df.columns generating the list of labels on an automatic way and then call it from the rename function. Some code optimization!! 
2. Eliminamos las columnas ['Unnamed_22', 'Unnamed_23'] puesto que no traen datos junto con las demás que traigan datos NaN en más de la mitad de sus registros. In the real world, you must be careful dropping null values, a previous analysis is needed in order to assess the more convenient way of dealing with this. For example, if you don't have enough records in your data set and you drop a column you could get unexpected and not desired results in modeling stage. For further explanations, come to me when you have a slot. 
3. In[12]: when you need to slice your dataframe by more than 2 or three columns try to use the iloc or loc pandas functions. It makes your code more readable. 
4. In[14]: Activity could be considered a categorical variable taking the values in your dataframe. The first step would be to convert the data type to category. Once you have changed the data type you can call, i.e. the cagtegories function and value_counts, to get the different categories we have in our dataset and the number of values we have in each category. This is very useful, for example, for bining.
5. In[20]: Take a look on the caveats Jupyter throws. This one is important "/home/alberto/.local/lib/python3.6/site-packages/pandas/core/indexing.py:543: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead". I encourage you to practise iloc/loc functions from now on. 
6. At the end of the file you comment "
Hemos seleccionado qcut porque nuestra distribución tiene muchos valores atípicos (outliers): USA y Australia muchos ataque pero la gran mayoría del resto muy pocos". Where is (in the file) the chunk of code that let us asserting that? Have you calculated the outliers in this file or maybe in another one in your laptop?